index.bs

```diff
@@ -1076,6 +1076,20 @@ with a [{{NoInterfaceObject}}] [=extended attribute=],
 and [=callback interfaces=] which declare [=constants=]
 must be annotated with an [{{Exposed}}] [=extended attribute=].
 
+<div algorithm>
+
+The <dfn>qualified name</dfn> of an [=interface=] |interface| is defined as follows:
+
+1.  Let |identifier| be the [=identifier=] of |interface|.
+1.  If |interface| has a [{{LegacyNamespace}}] [=extended attribute=], then:
+    1.  Let |namespace| be the identifier argument of the [{{LegacyNamespace}}]
+        [=extended attribute=].
+    1.  Return the [=concatenation=] of « |namespace|, |identifier| » with
+        separator <span class="char">U+002E FULL STOP (".")</span>.
```

Comment:
A bit unsure if (".") needs to be in the `<span class=char>` as well…

Reply:
Looking at the existing spec for about ten seconds yields
```html
<span class="char">U+005F LOW LINE ("_")</span>
```